docs/pushpayloads.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "title":"FxA Push Payload schema",
+  "description":"This schema defines what is acceptable to send as a payload data with the Push API from the FxA servers to a device connected to FxA",
+  "$schema":"http://json-schema.org/draft-04/schema#",
+  "type":"object",
+  "required":[
+    "version",
+    "data"
+  ],
+  "properties":{
+    "version":{
+      "type":"number",
+      "description":"The version of this push payload data instance. Bump this if you make changes to any part of this schema."
```

Comment:
I was just 1/2 way through writing a reply that came to the same basic conclusions; +1